tensorflow/lite/kernels/conv_test.cc

```diff
@@ -168,6 +170,37 @@ TEST_P(ConvolutionOpTest, SimpleTestFloat32) {
                              }));
 }
 
+TEST_P(ConvolutionOpTest, DISABLED_SimpleTestFloat32NumThread1) {
```

Comment:
Why disabled? Why not actually test the multi-threaded case?

Reply:
@jdduke : Sorry i think i uploaded my experimental code, will update it, Thanks for pointing out.